userweb/bower.json

```diff
@@ -20,7 +20,8 @@
     "angularjs-datepicker": "~0.2.4",
     "angular-i18n": "~1.3.15",
     "moment": "~2.10.2",
-    "crypto-js": "~3.1.4"
+    "crypto-js": "~3.1.4",
+    "angular-qr": "latest"
```

Comment:
We prefer avoid ultra dynamic versions for bower and node modules.

I would replace _latest_ with `~0.2.0` or `0.2.0` which seems the latest [angular-qr release](https://github.com/janantala/angular-qr/releases)